channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -313,6 +314,8 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                         _endpointUrl = _orgEndpointUrl = senderConfig.endpointUrl;
                     }
 
+                    _getStatsBeat(); // first call will initialize the statsbeat object
```

Comment:
This is going to be potentially problematic, while it does cause the configuration to get referenced so that it would get called again, because the result is not used it "may" actually get removed by the minification process (and therefore would never get referenced).

Looking at the implementation, however, my question is 
- do we really need to call this during initialization? As anytime it's first used it should get initialized?

Reply:
i.e. Is it "needed" for some reason that I'm not spotting atm.